propagation.go

```diff
@@ -114,6 +114,9 @@ func injectTraceContext(sc SpanContext, opaqueCarrier interface{}) error {
 	return nil
 }
 
+// This method searches for Instana headers (FieldT, FieldS, FieldL and header with name prefixed with FieldB)
+// and try to parse their values. It also tries to extract w3 context and assign it inside returned object. W3 context
```

Comment:
```suggestion
// and try to parse their values. It also tries to extract W3C context and assign it inside returned object. W3C context
```
C stands for consortium there 😉